docs/Researcher/best-practices/researcher-notifications.md

```diff
@@ -1,31 +1,25 @@
 ---
-title: Researcher Notifications
+title: Researcher Email Notifications
 summary: This article describes researcher notifications and how to configure them.
 authors:
     - Jason Novich
-    - Shiri   
+    - Shiri Arad
 date: 2024-Jul-4
 ---
 
-## AI Practitioners Email Notifications
+## Importance of Email Notifications for Data Scientists
 
-### Importance of Email Notifications for Data Scientists
-
-Managing numerous data science workloads requires monitoring various stages, including submission, scheduling, initialization, execution, and completion. Additionally, handling suspensions and failures is crucial for ensuring timely workload completion.
-Email Notifications, introduced in Run:ai version 2.18, address this need by sending alerts for critical workload life cycle changes. This empowers data scientists to take necessary actions and prevent delays.
+Managing numerous data science workloads requires monitoring various stages, including submission, scheduling, initialization, execution, and completion. Additionally, handling suspensions and failures is crucial for ensuring timely workload completion. Email Notifications address this need by sending alerts for critical workload life cycle changes. This empowers data scientists to take necessary actions and prevent delays.
 
 Once the system administrator configures the email notifications, users will receive notifications about their jobs that transition from one status to another. In addition, the user will get warning notifications before workload termination due to project-defined timeouts. Details included in the email are:
```

Comment:
Once the system administrator configures the email notifications, users will be able to set up personal notifications about their workloads that transition from one status to another. In addition, the user will get warning notifications before workload termination due to project-defined timeouts. Details included in the email are: